packages/react-core/src/components/AboutModal/examples/AboutModalComplexUserPositionedContent.tsx

```diff
@@ -24,12 +25,12 @@ export const AboutModalComplexUserPositionedContent: React.FunctionComponent = (
         hasNoContentContainer={true}
         productName="Product Name"
       >
-        <TextContent id="test1" className="pf-v5-u-py-xl">
+        <TextContent id="test1" className={spacing.pyXl}>
```

Comment:
I have an issue in org to improve our documentation around react-tokens and react-styles so that products can better utilize them. I'd personally advocate for demonstrating how to use them well since this will reduce the work required to migrate to new major versions of PF. This avoids the hardcoding of strings with versioned prefixes.